liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/BasicSupport.java

```diff
@@ -374,4 +390,56 @@ protected String stripVersionFromName(String name, String version) {
             return name;
         }
     }
+    
+    protected void installLicense() throws MojoExecutionException, IOException {
+        if (licenseArtifact != null) {
+            Artifact license = getArtifact(licenseArtifact);
+            if (!hasSameLicense(license)) {
+                log.info(MessageFormat.format(messages.getString("info.install.license"), licenseArtifact.getArtifactId()));
+                Java installLicenseTask = (Java) ant.createTask("java");
+                installLicenseTask.setJar(license.getFile());
+                Argument args = installLicenseTask.createArg();
+                args.setLine("--acceptLicense " + assemblyInstallDirectory.getCanonicalPath());
+                installLicenseTask.setTimeout(30000L);
+                installLicenseTask.setFork(true);
+                int rc = installLicenseTask.executeJava();
+                if (rc != 0) {
+                    throw new MojoExecutionException(MessageFormat.format(messages.getString("error.install.license"), rc));
+                }
+            }
+        }
+    }
+    
+    private boolean hasSameLicense(Artifact license) throws MojoExecutionException, IOException {
+        boolean sameLicense = false;
+        if (license != null) {
+            InputStream licenseInfo = getZipEntry(license.getFile(), "wlp/lafiles/LI_en");
+            if (licenseInfo == null) {
+                // TODO: The license file may be corrupted
+                log.warn("The license file may be corrupted");
```

Comment:
Should this be added to the messages file?  Maybe it intended to use warn.install.license after that is fixed in the properties file?